homeassistant/components/qbittorrent/__init__.py

```diff
@@ -1 +1,85 @@
 """The qbittorrent component."""
+import asyncio
+import logging
+
+from qbittorrent.client import LoginRequired
+from requests.exceptions import RequestException
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_URL, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import config_per_platform
+
+from .client import create_client
+from .const import DATA_KEY_CLIENT, DATA_KEY_NAME, DOMAIN
+
+PLATFORMS = ["sensor"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
```

Comment:
Is this logic is done inside sensor.py, you can removed the part of needing the handle the config per platform and trigger the import in `async_setup_platform` of the sensor instead.

(Which is also a nice place to throw a deprecation warning of the YAML configuration being deprecated).